src/main/java/org/fungover/storm/fileHandler/FileRequestHandler.java

```diff
@@ -0,0 +1,51 @@
+package org.fungover.storm.fileHandler;
+
+import org.fungover.storm.client.HttpParser;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+
+public class FileRequestHandler {
+    private static Path path;
+    private static byte[] file;
+    private static long filesize;
+
+    private FileRequestHandler(){}
+
+    public static void handleRequest(String request) throws IOException {
+        var map = HttpParser.getRequestHeaders(request);
+        if (map.get("path").equals("/"))
+            map.put("path", "index.html");
+        path = Paths.get(getAbsolutePathToResourceFromContext(map, System.getProperty("os.name")));
+        file = Files.readAllBytes(path);
+        filesize = file.length;
+    }
+
+
+    public static byte[][] writeResponse() {
+        String response = "HTTP/1.1 200 OK \r\nContent-length:" + filesize +
+                "Content-type:" + FormatConverter.MIME(path) +
```

Comment:
Missing new line between content-length och content-type.
```suggestion
                "\r\nContent-type:" + FormatConverter.MIME(path) +
```